pkg/rancher-desktop/components/Preferences/BodyVirtualMachine.vue

```diff
@@ -59,12 +59,18 @@ export default Vue.extend({
         },
       );
     },
+    selectTab(tab: string) {
+      const tabbed = this.$refs.tabbed as RdTabbed;
+
+      tabbed.$children[0].select(tab);
+    },
   },
 });
 </script>
 
 <template>
   <rd-tabbed
+    ref="tabbed"
```

Comment:
We can remove `ref` because we don't want to directly invoke child functions from the parent.